src/index.js

```diff
@@ -1,8 +1,31 @@
 import './style.css';
-import tasks from '../modules/create-tasks-array.js';
-import taskNode from '../modules/create-task-node.js';
+import tasks from '../modules/tasks-array-module.js';
+import { populateTaskArray } from '../modules/create-tasks-array.js';
+import displayNode, { toDoList } from '../modules/display-task-node.js';
 
-taskNode(tasks);
+const taskInput = document.querySelector('#add-new-task');
+const resetBtn = document.querySelector('#reset-button');
 
-// newTask.dataset.elementId = e.index;
-// newtask.dataset.completed = true;
+tasks.forEach((e) => {
+  displayNode(e);
+});
+
+taskInput.addEventListener('keydown', (event) => {
+  if (event.key === 'Enter') {
+    if (taskInput.value.length <= 1) {
+      return;
+    }
+    populateTaskArray(taskInput.value);
+    taskInput.value = '';
+  }
+});
+
+resetBtn.addEventListener('click', () => {
+  localStorage.clear();
+  tasks.splice(tasks[0]);
+
+  console.log(tasks);
+  while (toDoList.firstChild) {
+    toDoList.removeChild(toDoList.lastChild);
+  }
+});
```

Comment:
![req-missing-editing-task](https://user-images.githubusercontent.com/93039359/193271610-7426b855-9458-435f-be38-98038d34f071.png)


- [ ] You have added add & remove functionality :clap: but you didn't add the functionality for editing an exisitng task which is a crucial requirement of this milestone :clap: Kindly implement the functionality for editing the task :+1: